drivers/gles3/rasterizer_scene_gles3.cpp

```diff
@@ -2380,6 +2380,7 @@ void RasterizerSceneGLES3::render_scene(const Ref<RenderSceneBuffers> &p_render_
 	bool draw_sky = false;
 	bool draw_sky_fog_only = false;
 	bool keep_color = false;
+	bool draw_canvas = false;
```

Comment:
Would it make sense to instead have a more generic flag for `copy_framebuffer` giving the combinations of:
```
bg_mode: keep
 - keep_color: true
 - copy_framebuffer: false

bg_mode: canvas
 - keep_color: false
 - copy_framebuffer: true

bg_mode: camera_feed
 - keep_color: true
 - copy_framebuffer: true
```